### DISCUSSION TITLE
Can we have full application layouts?

### DISCUSSION THREAD

**Discussion by razb-viola** [shadcn-ui/ui#2479] [Q&A]
This is probably the best thing ever done in the world of frontend web app - this shadcn repository, really. 
You guys took all of that messy parts and just exposed to us a box of shiny brand new components with robust api to use!

Thanks for that really.

However I see that there non any component that help you build the application ui sidebar/header etc.
Also, some other useful components like stats are missing as well, take a look here:

https://tailwindui.com/components/application-ui/application-shells/sidebar
https://tailwindui.com/components/application-ui/data-display/stats

Perhaps you can work on add these type of components to shad as well?

Currently it feels that shad can really help with the ui kit, but not yet with the full foundation and the core of the app building blocks. Isn't it?

**Comment by DwennK** [comment]
yes
